### ISSUE TITLE
Error during startup 'ModuleNotFoundError: No module named 'libs.ettutv'

### ISSUE THREAD

**Issue by psiroky** [waladir/plugin.video.sportstreams#1]
I installed this add-on in version `1.0.4` (on Kodi Matrix 19.4, Raspberry PI 4) and I am getting the following error during start-up of the add-on (installation itself was successful):
```
...
ERROR <general>: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: <class 'ModuleNotFoundError'>
                                                   Error Contents: No module named 'libs.ettutv'
                                                   Traceback (most recent call last):
                                                     File "/storage/.kodi/addons/plugin.video.sportstreams/main.py", line 17, in <module>
                                                       from libs.ettutv import list_ettutv_main, list_ettutv_categories, list_ettutv_filter, play_ettutv_stream, get_ettutv_live_streams, list_ettutv_schedule
                                                   ModuleNotFoundError: No module named 'libs.ettutv'
                                                   -->End of Python script error report<--
...
```

When I list the contents on the `libs` directory of the add-on, there is no file `ettutv.py` (which I assume should be there).
```
# ls -lah /.kodi/addons/plugin.video.sportstreams/libs
total 59K    
drwxr-xr-x    3 root     root        1.0K Nov 14 13:13 .
drwxr-xr-x    5 root     root        1.0K Nov 14 13:13 ..
drwxr-xr-x    2 root     root        1.0K Nov 14 13:13 __pycache__
-rw-r--r--    1 root     root        6.2K Nov 14 13:13 ct4sportplus.py
-rw-r--r--    1 root     root       12.3K Nov 14 13:13 hustetv.py
-rw-r--r--    1 root     root        4.5K Nov 14 13:13 pingpongtv.py
-rw-r--r--    1 root     root       25.7K Nov 14 13:13 tvcomcz.py
-rw-r--r--    1 root     root         179 Nov 14 13:13 utils.py
-rw-r--r--    1 root     root        3.4K Nov 14 13:13 volejtv.py
```

Could it be the release 1.0.4 is missing that file? This commit https://github.com/waladir/plugin.video.sportstreams/commit/93e6c6195f95bcab1beca999b8a6ae3682fd52e3 seems to be adding support for [ETTU TV](https://www.ettu.tv/), but the `ettutv.py` is missing.

**Comment by waladir** [comment]
Thank you for reporting issue. It solved in 1.0.5 version.

**Comment by psiroky** [comment]
Thanks for a quick fix. The add-on now starts without any problem.
